templates/typescript/tsconfig.json.eta

```diff
@@ -28,7 +28,5 @@
     "metro.config.js",
     "jest.config.js"
   ],
-  <% if(it.expo) { %>
   "extends": "expo/tsconfig.base"
```

Comment:
Similarly, if this is the only condition in this file it could become a standard `.json` file.